templates/frag/extractMultipartValue.hbs

```diff
@@ -0,0 +1,11 @@
+{{!--
+Extract the correct value from the multipart input for use with the function from fromMultipart.
+@param <this> CodegenProperty or CodegenSchema
+--}}
+{{#if (isFile schema)}}
+value
+{{~else if (isMetadataSchema schema)~}}
```

Comment:
I think we should instead check `(isFile schema)`... there could be metadata schemas that aren't files.